plugins/plugin-webpack/plugin.js

```diff
@@ -200,8 +211,8 @@ module.exports = function plugin(config, args) {
         context: buildDirectory,
         resolve: {
           alias: {
-            "/__snowpack__": path.join(buildDirectory, "__snowpack__"),
-            "/web_modules": path.join(buildDirectory, "web_modules"),
+            [metaDir]: path.join(buildDirectory, "__snowpack__"),
```

Comment:
The ` "__snowpack__"` & ` "web_modules"` need to match the `webModulesDir` & `metaDir` above, as well. Otherwise, I'd assume that this code is currently failing.

Reply:
yes 😰

Reply:
fixed